libs/api/domains/cms/src/lib/search/contentful.service.ts

```diff
@@ -170,6 +170,19 @@ export class ContentfulService {
     return _.flatten(chunkedChanges)
   }
 
+  private async linksToEntry(
```

Comment:
Maybe calling this function `getParentEntries` might be more descriptive

Reply:
I decided to let the name reflect the Contentful API.

According to [Contentful documentation](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters/links-to-entry), this functionality has more possible use cases than we're using it for.
> To search for entries which have a field linking to a specific entry

I do agree this name is not very descriptive, when in use, but I think it's also good that it reflects their API.
Some other possible approaches, I would like to take up for discussion:
- create `getParentEntries` as an alias function of `linksToEntry`, which would let code "document/describe" it self
- comment linksToEntry, with what it does
- comment when use, what you're trying to achieve
- just rename the function like you suggest, because it probably won't be used in any other way.

any thoughts?

Reply:
Well our code does not reflect the Contentful API already e.g. `getContentfulData` is not called `getEntriesByIds`.
But I don't see this as a big issue, I'm ok with this either way